### ISSUE TITLE
Add spec

### ISSUE THREAD

**Issue by jchouse** [jchouse/csscomb-atom#20]
Will leave it here

**Comment by rugk** [comment]
What exactly do you mean by "spec"?

**Comment by jchouse** [comment]
specification

**Comment by rugk** [comment]
Yeah, but what specification do you want to add here? I mean this is no protocol or something similar.

**Comment by jchouse** [comment]
Just the documentation to bring in, normalneho condition

**Comment by rugk** [comment]
So you mean inline documentation in your JS source?
